### ISSUE TITLE
Documentation about New Search And How to Use It

### ISSUE THREAD

**Issue by bbenetskyy** [mgierlasinski/MagicGradients#36]
Just update Docs and provide in short and clear way how it can be used.

📛 This issue is blocked by #35 & #34 📛

**Comment by mgierlasinski** [comment]
@bbenetskyy Any update on this? This looks like duplicate of https://github.com/mgierlasinski/MagicGradients/issues/67 for me.

**Comment by bbenetskyy** [comment]
Not a duplicate, because it's more about adding a description about a new feature and in #67 going more about re-writing existing documentation, this could be as a part of task #67 but it's not required. 

I think we will be sure when starting the implementation of #67 😉

**Comment by mgierlasinski** [comment]
@bbenetskyy Do you play to do anything with that? If not, please close.
